mne/time_frequency/tests/test_tfr.py

```diff
@@ -212,8 +212,8 @@ def test_time_frequency():
 def test_dpsswavelet():
     """Test DPSS wavelet"""
     freqs = np.arange(5, 25, 3)
-    Ws = _dpss_wavelet(1000, freqs=freqs, n_cycles=freqs / 2.,
-                       time_bandwidth=4.0, zero_mean=True)
+    Ws = dpss_wavelet(1000, freqs=freqs, n_cycles=freqs / 2.,
```

Comment:
`morlet` was public and dpss_wavelet not so either both should be private or both public.